README.md

```diff
@@ -1 +1,5 @@
-Hooray! I am going to learn Git now!
+Hooray! We are going to work on learning Git now!
```

Comment:
We like short code. Let's use a contraction instead of "we are".

Reply:
Updated

Reply:
Thanks!